package.json

```diff
@@ -7,9 +7,27 @@
     "start": "astro dev",
     "build": "astro build",
     "preview": "astro preview",
-    "astro": "astro"
+    "astro": "astro",
+    "prepare": "husky install"
   },
   "dependencies": {
     "astro": "^2.9.3"
+  },
+  "devDependencies": {
+    "@typescript-eslint/parser": "^6.2.0",
+    "husky": "^8.0.3",
+    "lint-staged": "^13.2.3",
+    "postcss-html": "^1.5.0",
+    "prettier": "^3.0.0",
+    "prettier-plugin-astro": "^0.11.0",
+    "stylelint-config-html": "^1.1.0"
+  },
+  "lint-staged": {
+    "*.astro": [
+      "prettier --write"
```

Comment:
this one blocks commits. Please add another ticket to find a way to lint those `*.astro` files. 
For now this should be removed